server/apps/immich/src/api-v1/job/job.service.ts

```diff
@@ -115,4 +138,8 @@ export class JobService {
         throw new BadRequestException(`Invalid job id: ${jobId}`);
     }
   }
+
+  private getFileNameWithoutExtension(path: string): string {
+    return basename(path, extname(path));
+  }
```

Comment:
We seem to have this or similar code scattered through the repo, we should move it to a util function instead.